contrib/apiage.py

```diff
@@ -388,7 +395,9 @@ def main():
             "fix-versions",
             "find-updates",
             "updates-to-markdown",
+            "updates-to-issuetemplate",
             "promote",
+            "next-ver",
```

Comment:
What is the purpose of new `next-ver` mode?

Reply:
It is unused for this PR. I will remove it.

Reply:
Removed.